TestApp/app/screens/AppCenterScreen.js

```diff
@@ -320,8 +320,9 @@ export default class AppCenterScreen extends Component {
   }
 }
 
-function runMBaaSCode() {
+async function runMBaaSCode() {
```

Comment:
`MBaaS` naming isn't specific enough, how about this?
```suggestion
async function runDataCrudScenarios() {
```

Reply:
Changed